RecoBTag/ONNXRuntime/python/pfDeepBoostedJetTags_cfi.py

```diff
@@ -0,0 +1,40 @@
+from RecoBTag.ONNXRuntime.SwitchProducerONNX import SwitchProducerONNX
+from RecoBTag.ONNXRuntime.deepBoostedJetONNXJetTagsProducer_cfi import deepBoostedJetONNXJetTagsProducer
+from RecoBTag.MXNet.boostedJetMXNetJetTagsProducer_cfi import boostedJetMXNetJetTagsProducer
+from RecoBTag.ONNXRuntime.Parameters.DeepBoostedJet.V02.pfDeepBoostedJetPreprocessParams_cfi import pfDeepBoostedJetPreprocessParams
+from RecoBTag.ONNXRuntime.Parameters.DeepBoostedJet.V02.pfMassDecorrelatedDeepBoostedJetPreprocessParams_cfi import pfMassDecorrelatedDeepBoostedJetPreprocessParams
+
+_flav_names = ['probTbcq', 'probTbqq', 'probTbc', 'probTbq', 'probWcq', 'probWqq', 
+               'probZbb', 'probZcc', 'probZqq', 'probHbb', 'probHcc', 'probHqqqq', 
+               'probQCDbb', 'probQCDcc', 'probQCDb', 'probQCDc', 'probQCDothers']
```

Comment:
this is apparently a copy from ```deepBoostedJetONNXJetTagsProducer.flav_names```
https://github.com/cms-sw/cmssw/blob/d8951bbd2766485a7845a36b120d61156b144ed9/RecoBTag/ONNXRuntime/plugins/DeepBoostedJetONNXJetTagsProducer.cc#L143-L161
why an explicit copy is needed here?